scripts/logfetch/grep.py

```diff
@@ -2,18 +2,21 @@
 import sys
 from termcolor import colored
 
-GREP_COMMAND_FORMAT = 'xargs -n {0} {1} < {2}'
+GREP_COMMAND_FORMAT = 'while read file; do {0} "$file"; done < {1}'
```

Comment:
More complicated custom grep commands weren't playing nicely with xargs. It was more flexible to loop through the file so any grep command that is used will output the way you'd expect it to.